### PR TITLE
Fix NPE made by PR 8865d4a

### PR DESCRIPTION
**Pull request by valib**
It stops working the HTTP engine V1. Reported in #2612. See the log.

### CODE REVIEW

File: src/main/java/net/pms/network/RequestHandler.java [hunk 1]
```diff
@@ -104,17 +104,8 @@ public void run() {
 			// header matches are attempted and if those fail as well we're stuck with the
 			// default renderer.
 
-			// Attempt 1: If the reguested url contains the no-transcode tag, force
-			// the default streaming-only conf.
-			if (request.getArgument().contains(RendererConfiguration.NOTRANSCODE)) {
```
Comment (valib) [inline src/main/java/net/pms/network/RequestHandler.java:109]: `request` is always null at this line.

Comment (SubJunk) [reply src/main/java/net/pms/network/RequestHandler.java:109]: @valib cool, can you do that as part of this branch, or should I?

Comment (valib) [reply src/main/java/net/pms/network/RequestHandler.java:109]: @SubJunk it would be better if you do it

Comment (SubJunk) [reply src/main/java/net/pms/network/RequestHandler.java:109]: That's done now, can you review?

**Comment by SubJunk** [comment]
@valib I'm going to do a release in about 12 hours so if this can be merged by then it would be good

**Review by SubJunk — APPROVED** [review]

**Pull request merged**